tests/jax/integration/parameter_server_test.py

```diff
@@ -72,11 +68,17 @@ def test_parameter_server_single_process(test_system_sp: System) -> None:
     assert list(trainer_steps) == [1]  # trainer.step one time
     assert list(executor_episodes) == [3]  # run episodes three times
 
-    network = parameter_server.get_parameters("networks-network_agent")
-    for network in network.values():
-        assert sorted(list(network.keys())) == ["b", "w"]
-        assert jnp.size(network["w"]) != 0  # network is updated
-        assert jnp.size(network["b"]) != 0
+    # check that the network is updated (at least one of the values updated)
+    updated_network = parameter_server.get_parameters("networks-network_agent")
+    at_least_one_changed = False
+    for key in updated_network.keys():
+        assert sorted(list(updated_network[key].keys())) == ["b", "w"]
+        if not jnp.array_equal(
+            updated_network[key]["w"], first_network[key]["w"]
+        ) or not jnp.array_equal(updated_network[key]["b"], first_network[key]["w"]):
```

Comment:
first_network[key]["b"]

Reply:
this is the old network before it gets updated it has the same keys as the updated_network

Reply:
I think he is referring to the typo on line 78. It should be `first_network[key]["b"]`.

Reply:
yeah but we are comparing biases to weights here 
or not jnp.array_equal(updated_network_param[key]["b"], **first_network_param[key]["b"])**:

Reply:
Thanks